games-emulation/jrommanager/jrommanager-1.6-r1.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN=JRomManager
+MY_PV=1.6b20
+MY_P="${MY_PN}-${MY_PV}"
+
+inherit desktop
+
+DESCRIPTION="A Mame and Retrogaming Rom Manager, Java alternative to ClrMamePro"
+HOMEPAGE="https://github.com/optyfr/JRomManager"
+SRC_URI="https://github.com/optyfr/${MY_PN}/releases/download/${MY_PV}/${MY_P}.zip -> ${P}.zip"
+QA_PREBUILT="*"
+
+S="${WORKDIR}"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND=">=virtual/jre-1.8"
+DEPEND="${DEPEND}"
```

Comment:
Please add unzip here.

Reply:
It was there in the initial commit of this pull request, then @zlogene told me to remove it, I asked him if zip format was automatically handled by 'unpack' but never got answered...

Reply:
Well I was once told to include it, I think it shouldn't hurt.

Reply:
But if @Zlogene tells you to omit it, please listen to him :) Please ping me then so I'll know for the next time.

Or if you can, use tar.gz which should be smaller and then you don't have to include anything in DEPEND.

Reply:
@hydrapolic I was not talking that `app-arch/unzip` is not needed to unpack a zip archive. But I was talking that upstream has the tar archived sources

Reply:
OK!!! So finally app-arch/unzip is needed, as I repeat again the upstream (that's me) has a tar archived sources but it can't be compiled without using at least gradle 4.7 that does not exists on gentoo and is officially not usable within portage (and gradle-wrapper is even more not usable)

Reply:
app-arch/unzip has now been added